python/ray/services.py

```diff
@@ -1053,6 +1057,8 @@ def start_dashboard(require_dashboard,
         f"--redis-address={redis_address}",
         f"--temp-dir={temp_dir}",
     ]
+    if append_log_dir:
+        command += [f"--log-dir={append_log_dir}"]
```

Comment:
Is it better to set the `log_dir` to None when `"RAY_USE_NEW_DASHBOARD" not in os.environ`? Then we can:
``` python
if log_dir:
    command += [f"--log-dir={log_dir}"]
```

Reply:
LGTM